db/objects/job/job_notifications.sql

```diff
@@ -1,13 +1,13 @@
 DO LANGUAGE PLpgSQL $block$
 BEGIN
-    IF reclada_object.list('{"class": "Message", "attrs": {"class": "Job", "event": "create"}}') is null THEN
-        PERFORM reclada_object.create('{"class": "Message", "attrs": {"channelName": "job_created", "class": "Job", "event": "create", "attrs": ["status", "type"]}}'::jsonb);
+    IF reclada_object.list('{"class": "Message", "attributes": {"class": "Job", "event": "create"}}') is null THEN
+        PERFORM reclada_object.create('{"class": "Message", "attributes": {"channelName": "job_created", "class": "Job", "event": "create", "attrs": ["status", "type"]}}'::jsonb);
```

Comment:
```suggestion
        PERFORM reclada_object.create('{"class": "Message", "attributes": {"channelName": "job_created", "class": "Job", "event": "create", "attributes": ["status", "type"]}}'::jsonb);
```
Do we need it? Evgeniy said that it works well for now. @AndreyRuma could you approve it?

Reply:
This is not our code. I think that it might have been changes on other branches.